changelog/README.md

```diff
@@ -46,7 +46,7 @@
 ### 3.7.1
 
 - [bug] JAVA-2174: Metadata.needsQuote should accept empty strings.
-
+- [bug] JAVA-2193: Fix flaky tests in WarningsTest.
```

Comment:
Where does this come from?

Reply:
It was deleted in a previous commit and re-added in this one, there are no changes.

Reply:
Please squash it into the previous commit that removes it.